cmd/swarmctl/main.go

```diff
@@ -13,15 +13,19 @@ import (
 	"github.com/docker/swarmkit/cmd/swarmd/defaults"
 	"github.com/docker/swarmkit/version"
 	"github.com/spf13/cobra"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 func main() {
 	if c, err := mainCmd.ExecuteC(); err != nil {
-		c.Println("Error:", grpc.ErrorDesc(err))
+		msg := err.Error()
```

Comment:
Non-blocking:  This works fine, but from the GRPC docstring for `status.FromError`:  

> FromError returns a `Status` representing err if it was produced from this package or has a method `GRPCStatus() *Status`. Otherwise, ok is false and a Status is returned with codes.Unknown and the original error message.

So it sounds like you can probably just do:

```golang
s, _ := status.FromError(err)
c.Println(s.Message())
```